recipes/physher/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "physher" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('physher', max_pin="x") }}
+
+source:
+  url: https://github.com/4ment/physher/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 89de0789af1136ca9a08f009b1db45d441bcb1d4ea7519103a998916b37c618f
+
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+  host:
+    - gsl
+  run:
+    - gsl
```

Comment:
```suggestion
```

gsl has run-exports defined: https://github.com/conda-forge/gsl-feedstock/blob/main/recipe/meta.yaml#L17

So this is not needed, it will be added automatically.